### PR TITLE
Onboarding design

### PR DESCRIPTION
**Pull request by lieblichaaron**
## ℹ️ Change description:
Nothing here

<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

### CODE REVIEW

File: locomotion-sdk/src/context/onboarding/index.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const authContainer = () => {
 
       if (vertResponse.status !== 'OK' || !vertResponse.refreshToken || !vertResponse.accessToken) {
         console.log('Bad vert with response', vertResponse);
-        return false;
+        return 'failed';
```
Comment (avivbarniv) [inline locomotion-sdk/src/context/onboarding/index.js]: Why you changed it?

Comment (lieblichaaron) [reply locomotion-sdk/src/context/onboarding/index.js]: need to check that it returned false to show error but even if theres no error the function returns nothing so it came back false and showed error, so i switched it to check if returns failed

File: locomotion-sdk/src/pages/AuthScreens/Onboarding/Name.js [hunk 2]
```diff
@@ -31,15 +31,26 @@ const Name = () => {
         />
         <TextInput
```
Comment (avivbarniv) [inline locomotion-sdk/src/pages/AuthScreens/Onboarding/Name.js:31]: I think it's better to create a base/generic input field and manage the focused style there, we will use it all across the app

File: locomotion-sdk/src/pages/AuthScreens/Onboarding/Phone.js [hunk 3]
```diff
@@ -8,11 +8,13 @@ import { ErrorText, PageContainer, SafeView } from './styles';
 import Header from './Header';
 import ScreenText from './ScreenText/index';
 import { loginApi } from '../../../context/user/api';
+import { ERROR_COLOR } from '../../../services/sharedStyles';
 
 const Phone = () => {
   const { onboardingState, updateState } = onboardingContext.useContainer();
   const navigation = useNavigation();
   const [showErrorText, setShowErrorText] = useState(false);
+  const [isFocused, setIsFocused] = useState(false);
```
Comment (avivbarniv) [inline locomotion-sdk/src/pages/AuthScreens/Onboarding/Phone.js]: same as above

File: locomotion-sdk/src/pages/AuthScreens/Onboarding/ScreenText/styles.js [hunk 4]
```diff
@@ -3,16 +3,16 @@ import styled from 'styled-components';
 export const TextContainer = styled.View`
 display: flex;
 flex-direction: column;
-margin-bottom: 20px;
+margin-bottom: 40px;
 `;
 
 export const Text = styled.Text`
-font-size: 16px;
+font-size: 18px;
```
Comment (avivbarniv) [inline locomotion-sdk/src/pages/AuthScreens/Onboarding/ScreenText/styles.js]: use const of font sizes

File: locomotion-sdk/src/pages/AuthScreens/Onboarding/ScreenText/styles.js [hunk 5]
```diff
@@ -3,16 +3,16 @@ import styled from 'styled-components';
...
 font-weight: 500;
 margin: 5px 0;
 `;
 
 export const SubText = styled.Text`
-font-size: 14px;
+font-size: 15px;
```
Comment (avivbarniv) [inline locomotion-sdk/src/pages/AuthScreens/Onboarding/ScreenText/styles.js]: use const of font sizes

File: locomotion-sdk/src/pages/AuthScreens/Onboarding/Welcome/styles.js [hunk 6]
```diff
@@ -1,26 +1,16 @@
...
-`;
-
 export const WelcomeText = styled.Text`
 font-size: 35px;
 font-weight: 600;
 margin: 10px 0;
 text-align: center;
 `;
 export const WelcomeSubText = styled.Text`
+font-size: 14px;
```
Comment (avivbarniv) [inline locomotion-sdk/src/pages/AuthScreens/Onboarding/Welcome/styles.js]: use font const

File: locomotion-sdk/src/pages/AuthScreens/StartScreen/index.js [hunk 7]
```diff
@@ -56,16 +64,28 @@ const StartScreen = ({ navigation }) => {
...
                 onPress={() => {
                   navigation.navigate('Phone');
                 }}
               >
-                <ButtonText>{i18n.t('login.getStarted')}</ButtonText>
+                <ButtonText dark>{i18n.t('login.signUp')}</ButtonText>
+              </StartButton>
+              <StartButton
+                onPress={() => {
+                  navigation.navigate('Phone');
```
Comment (avivbarniv) [inline locomotion-sdk/src/pages/AuthScreens/StartScreen/index.js:85]: use the navigation service

**Review by avivbarniv — APPROVED** [review]

File: locomotion-sdk/src/context/onboarding/index.js (hunk 1 above)
Comment (avivbarniv) [reply locomotion-sdk/src/context/onboarding/index.js]: So add return true in the end of the function

File: locomotion-sdk/src/pages/AuthScreens/StartScreen/index.js (hunk 7 above)
Comment (lieblichaaron) [reply locomotion-sdk/src/pages/AuthScreens/StartScreen/index.js:85]: you mean create a navigation service?

File: locomotion-sdk/src/pages/AuthScreens/Onboarding/Phone.js (hunk 3 above)
Comment (lieblichaaron) [reply locomotion-sdk/src/pages/AuthScreens/Onboarding/Phone.js]: i cant put this in the component its a dependency

File: locomotion-sdk/src/pages/AuthScreens/StartScreen/index.js (hunk 7 above)
Comment (lieblichaaron) [reply locomotion-sdk/src/pages/AuthScreens/StartScreen/index.js:85]: i made a new branch and created a navigation service. ill switch over the all the uses there

File: locomotion-sdk/src/Components/PhoneNumberInput/codes.json [hunk 8]
```diff
@@ -1,982 +1,982 @@
 [
```
Comment (avivbarniv) [inline locomotion-sdk/src/Components/PhoneNumberInput/codes.json:1]: Why aren't we using a lib that gives us the flags?

Comment (lieblichaaron) [reply locomotion-sdk/src/Components/PhoneNumberInput/codes.json:1]: we are, the file was already there, im just using it now to get the code based on the default country

File: locomotion-sdk/src/services/sharedStyles.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
+export const ERROR_COLOR = '#f35657';
+
+export const OPERATION_COLOR = '#24aaf2';
+
+export const FONT_STYLES = {
+  SIZE: {
+    HEADER_1: '20px',
```
Comment (avivbarniv) [inline locomotion-sdk/src/services/sharedStyles.js]: set
small
medium
large
extra_large

**Review by avivbarniv — APPROVED** [review]

File: locomotion-sdk/src/services/sharedStyles.js (hunk 9 above)
Comment (lieblichaaron) [reply locomotion-sdk/src/services/sharedStyles.js]: but what if we use sizes in between? i switched it to names so we could leave spots for potentially other sizes. what would you call 14px if small and medium are already there?

**Pull request merged**